src/storage/src/hummock/sstable/multi_builder.rs

```diff
@@ -154,7 +154,7 @@ where
         }
 
         let builder = self.current_builder.as_mut().unwrap();
-        builder.add(full_key, value, is_new_user_key).await
+        builder.add(*full_key, value, is_new_user_key).await
```

Comment:
Can we remove make the parameter type of `full_key` into `FullKey<&[u8]>` without a reference by the way? And then here we can pass `full_key` directly.